pkg/ebpf/testdata/run_latency_bench.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -e
+
+if [ "${EUID}" != "0" ]
+then
+  echo "Error: this command must be run as root" >&2
+  exit 1
+fi
+
+if ! [ -x "$(command -v benchstat)" ]
+then
+  echo "Error: benchstat could not be found" >&2
+  exit 1
+fi
+
+now=$(date -u +"%Y-%m-%dT%H%M%S")
+RESFILE=/tmp/ebpf-bench-results.${now}
+STATFILE=/tmp/ebpf-bench-benchstat.${now}
+
+# using the test.count flag doesn't let us report probe times for each iteration
+# use bash loop instead
+for _ in {1..5}
+do
+  go test \
+    -mod=vendor \
+    -tags linux_bpf,ebpf_bindata \
+    ./pkg/ebpf/... \
+    -bench BenchmarkTCPLatency \
+    -count=1 \
+    -run XXX \
+    >> "${RESFILE}"
+  go test \
+    -mod=vendor \
+    -tags linux_bpf,ebpf_bindata \
+    ./pkg/ebpf/... \
+    -bench BenchmarkUDPLatency \
+    -count=1 \
+    -run XXX \
```

Comment:
What is `-run XXX` doing ?

Reply:
That prevents it from running any tests, since `XXX` doesn't match any test names.

Reply:
I'd add a comment to that effect, since I don't think this is a common pattern.

Reply:
It is definitely a common pattern. Some folks use `'^$'` to prevent matching, but regardless if you only want to run benchmarks you need to prevent test matches.

Here are a couple articles that use a similar pattern:
https://scene-si.org/2017/06/06/benchmarking-go-programs/
https://medium.com/@hackintoshrao/daily-code-optimization-using-benchmarks-and-profiling-in-golang-gophercon-india-2016-talk-874c8b4dc3c5